resample_states_verification/NOTES.md

```diff
@@ -0,0 +1,58 @@
+
+Testing of changes to use mock GET_PITCH commands
+=================================================
+
+Used the timelines code to make a new test database and cmd states for 2010.  This made
+the archive by stepping through the days, which we don't care about for this, but, in the
+timelines repo:
+
+
+    import timelines_test
+    timelines_test.all_2010(cmd_state_ska='/data/fido/ska')
+
+I then used fetch_states against the flight cmd_states.h5 and the testing cmd_states.h5 to
+create dat files limited to the columns that really should not be changing due to this
+update.
+
+    from Chandra.cmd_states.get_cmd_states import fetch_states
+    from astropy.table import Table
+    
+    official = fetch_states(start='2010:005', stop='2010:360', vals=['obsid', 'power_cmd',
+      'si_mode', 'pcad_mode', 'vid_board', 'clocking', 'fep_count', 'ccd_count', 'simpos',
+      'simfa_pos', 'hetg', 'letg'], server='/proj/sot/ska/data/cmd_states/cmd_states.h5')
+    Table(official).write('official_minus_point.dat', format='ascii')
+    
+    test = fetch_states(start='2010:005', stop='2010:360', vals=['obsid', 'power_cmd',
+      'si_mode', 'pcad_mode', 'vid_board', 'clocking', 'fep_count', 'ccd_count', 'simpos',
+      'simfa_pos', 'hetg', 'letg'], server='t/all_2010/cmd_states.h5')
+    Table(test).write('test_minus_point.dat', format='ascii')
+    
+    meld official_minus_point.dat test_minus_point.dat
+
+The small set of small timing diffs are acceptable on the non-pointing-related states.
```

Comment:
Ah.  That kind of detail.  Thanks.

Do we have a place to put test outputs that makes some sense?
 $SKA/data/task/test_outputs ?  Tagged somehow?  For small-ish files on
somewhat complicated test (that are somewhat non-trivial to make), it seems
worth keeping them for a bit.

> You can reliably predict I will object every time large regression test
files are included in a source repo.

Though I thought I *might* get away with it now that github is just better
about not trying to print everything, and since these aren't intended to be
part of some new process of regression testing.  Will remove and rebase etc.
​